config/manager/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+resources:
+- manager.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: mongodb-kubernetes-operator
+  newName: quay.io/mongodb/mongodb-kubernetes-operator:0.5.0
```

Comment:
it looks like the `newName` here will be what is substituted in the resulting deployment, we should make sure that we update our release steps with this extra file we need to change (until the whole thing is automated!)

Reply:
This will need a fairly large number of documentation changes around build/deploy process as well. I think maybe this should be part of a separate PR, if only to avoid making this PR any bigger 😅

Reply:
Ah okay, if you could create a ticket we can do that seperately.

Reply:
Done!